resources/robotino.rossystem

```diff
@@ -1,38 +1,41 @@
-RosSystem { Name 'robotino' 
-    RosComponents ( 
-        ComponentInterface { name 'joy'
+RosSystem { Name 'robotino'
+    RosComponents (
+        ComponentInterface { name 'joy_node' FromRosNode 'joy'
```

Comment:
This is not valid, the node ''joy' doesn't exists.

See please: https://github.com/ipa-rwu/ros_model_parser/pull/1/files